apps/api/src/server/swagger.js

```diff
@@ -89,6 +89,9 @@ export const spec = {
 			documents: [{ guid: '0084b156-006b-48b1-a47f-e7176414db29' }],
```

Comment:
Comment only:  when I add new definitions to the swagger file, I have been following Ben's lead and adding them to the "@definitions" section later in the doc (line 1116 onwards), rather than the "definitions" section, with much more explicit definition, as this allows us to define type, description, example, etc.